### PR TITLE
Add and correct view modes for rendering pages and venue for search indexing

### PR DESCRIPTION
**Pull request by ekes**
Pages was using the default template, which could have been causing content to get missed.
Venue was good, but included the titles of the fields that's just repeated words not related to the content, and rendered some things unnecessarily.

Helps #73

### CODE REVIEW

**Comment by ekes** [comment]
Oh! Is this still not a thing. Adding @stephen-cox @Adnan-cds @andybroomfield for review then!

Sorry I was just going through open PR's and saw this was sitting here.

File: modules/localgov_directories_venue/config/install/core.entity_view_display.node.localgov_directories_venue.directory_index.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+uuid: 4d6ed028-e23c-4552-b9fc-f9103df429fa
```
Comment (stephen-cox) [inline modules/localgov_directories_venue/config/install/core.entity_view_display.node.localgov_directories_venue.directory_index.yml]: Don't think this should be here

File: modules/localgov_directories_venue/config/install/core.entity_view_display.node.localgov_directories_venue.directory_index.yml [hunk 2]
```diff
@@ -18,9 +19,10 @@ dependencies:
     - node.type.localgov_directories_venue
   module:
     - link
-    - telephone
     - text
     - user
+_core:
+  default_config_hash: yuSNmUeMHlWYAbP3Wezm7Ijgp8IkvQa9TREh5dYZ-qI
```
Comment (stephen-cox) [inline modules/localgov_directories_venue/config/install/core.entity_view_display.node.localgov_directories_venue.directory_index.yml]: Should be remove as well

**Review by stephen-cox — CHANGES_REQUESTED** [review]
Couple of IDs to remove, but otherwise looks fine just from reviewing the config.

**Comment by ekes** [comment]
So changes are now only hiding the labels from the exisiting indexd view mode for venue, and adding the equivalent to the page.

**Comment by Adnan-cds** [comment]
I am happy for it to be merged once Steve approves it.

**Review by stephen-cox — APPROVED** [review]
I'm happy - let's merge

**Review by finnlewis — APPROVED** [review]
This looks good to me.

**Pull request merged**